doc/api/readline.md

```diff
@@ -412,6 +412,8 @@ changes:
   * `output` {stream.Writable} The [Writable][] stream to write readline data
     to.
   * `completer` {Function} An optional function used for Tab autocompletion.
+  * `postprocessor` {Function} An optional function used to post-process the
+    current line before writing it to `output`.
```

Comment:
This option will only work with `terminal === true`, right?

Reply:
correct